### PR TITLE
[ADD] website_sale_wishlist_keep: New module that keep the items on wishlist when we add to cart

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT24400

Please review this. @sergio-teruel @carlosdauden

### CODE REVIEW

**Review by HaraldPanten — APPROVED** [review]
IMO it's working as expected. Functional review 👍

File: website_sale_wishlist_keep/demo/assets.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="assets_editor_demo" inherit_id="website.assets_editor">
+        <xpath expr=".">
+            <script type="text/javascript"
+                    src="/website_sale_wishlist_keep/static/src/js/tour.js"/>
+        </xpath>
+    </template>
+</odoo>
```
Comment (sergio-teruel) [inline website_sale_wishlist_keep/demo/assets.xml]: This must be inside of views folder

File: website_sale_wishlist_keep/tests/test_ui.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+class TestUi(HttpCase):
+    def setUp(self):
+        super().setUp()
+        self.env["product.template"].create(
+            {
+                "name": "Test Product",
+                "website_published": True,
+                "website_sequence": 5000,
+                "type": "consu",
+                "inventory_availability": "always",
```
Comment (sergio-teruel) [inline website_sale_wishlist_keep/tests/test_ui.py]: Remove this field, this module does not denpeds on website_sale_stock

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by CarlosRoca13** [comment]
@sergio-teruel changes done!

**Review by sergio-teruel — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**